Notebooks/Pluto/p1.jl

```diff
@@ -52,13 +37,11 @@ To understand that, we must understand a few things about classical computers.
 
 \"*A bit (short for binary digit) is the smallest unit of data in a computer*\". You must have read this in a book or article about computers. Computers use bits, represented by the digits 0 and 1, to store data. We organize these bits to store information and manipulate these bits and perform operations on them to get a variety of things done. 
 Like storing numbers, or images or videos. Adding numbers, subtracting numbers.
-How? Well... Consider storing a number. In this case, the collection of bits 1 0 1 1 can be considered as, $ 2^3 + 0 + 2^1 + 2^0 = 11 $. The genereal idea being, the summation of $ 2^{position of 1s(starting from 0) from right} $.  
+How? Well... Consider storing a number. In this case, the collection of bits 1 0 1 1 can be considered as, $ 2^3 + 0 + 2^1 + 2^0 = 11 $. The genereal idea being, the summation of $ 2^{(position\; of\; 1s(starting\; from\; 0)\; from\; right)} $.  
```

Comment:
for consistency make the above bits as a math expression or just use an inline code block like `bit"1011"`?

Reply:
I don't recommend showing Yao code to the reader in the first 3 notebooks.
That'll just confuse them😅
The use of bits" " is properly shown later.

Reply:
@Roger-luo 
md"
 - When the address qubits give `` 00 `` or `` 0 `` in decimal, for the next 5 qubits, we get `` $a ``.
"
Doesn't work